src/helpers/array.js

```diff
@@ -260,3 +260,80 @@ export function arrayUnique(array) {
 
   return unique;
 }
+
+/**
+ * Differences in two or more array.
```

Comment:
This also might need to be more descriptive. I don't know if it's clear enough, that for more than 2 arrays, it will return the element that's not in _any_ of the arrays.